lib/resolveurl/plugins/cda.py

```diff
@@ -55,6 +55,15 @@ def cda_decode(self, url):
         strurl = ''
         for char in url:
             strurl += chr(33 + (ord(char) + 14) % 94) if 32 < ord(char) < 127 else char
+        strurl = self.da(strurl)
         strurl = strurl if strurl.startswith('http') else 'https://{0}'.format(strurl)
         strurl = strurl if strurl.endswith('.mp4') else '{0}.mp4'.format(strurl)
-        return strurl.replace("adc.mp4", ".mp4")
+        return strurl
+
+    def da(self, a):
+        a=a[:-4]
+        a = a.replace("0)sss.mp4", ".mp4")
```

Comment:
Are you sure that this line is required? I can't see any replace instance in the javascript code